app/src/main/java/com/door43/translationstudio/tasks/UpdateAppTask.java

```diff
@@ -149,6 +147,8 @@ private void updateBuildNumbers() {
      */
     private void upgradePre115() {
         AppContext.context().deleteDatabase(Library.DATABASE_NAME);
+        TargetTranslation[] targetTranslations = AppContext.getTranslator().getTargetTranslations();
+        TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslations);
```

Comment:
@neutrinog - But I don't think we want to make chunk migration dependent on package_version.